Cogs/StudentCommands.py

```diff
@@ -42,7 +42,7 @@ async def corgme(self, ctx, number=-1):
         images = ['dogs/corgis/' + path.name for path in Path('dogs').rglob('*.*')]
 
         # Generates a random number if no number is given
-        if number < 0:
+        if number < 0 or number > 99:
```

Comment:
This should be changed to `number > len(images)`

Reply:
Fixed with new commit.